development/dev-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nutsfoundation/nuts-node:latest as nutsnode
+FROM debian:buster-slim
```

Comment:
is it possible to use alpine here? This would prevent distro specific issues.

Reply:
Unfortunately the tunnel install script uses `apt-get`